doc/src/manual/documentation.md

```diff
@@ -3,7 +3,7 @@
 Julia enables package developers and users to document functions, types and other objects easily
 via a built-in documentation system.
 
-The basic syntax is simple: any string appearing at the toplevel right before an object
+The basic syntax is simple: any string appearing just above an object
```

Comment:
I think "before" should be kept since it doesn't have to be on the line above
```suggestion
The basic syntax is simple: any string appearing just before an object
```

Reply:
You are the one who knows. I am a newbie.